src/subdomains/staking/application/services/liquidity-management.service.ts

```diff
@@ -78,15 +79,19 @@ export class LiquidityManagementService {
 
   // --- MASTERNODES ---- //
   private async createMasternodes(count: number): Promise<void> {
+    const idleMasternodes = await this.masternodeService.getIdleMasternodes(count);
```

Comment:
The comment from line 52 should be here or?

Reply:
done!